wheel_odom:
  wheelbase: 1.55
  wheel_radius: 0.05
  odom_frame: odom
  base_frame: base_link
  odom_topic: /wheel_odom
  publish_tf: false
  body_forward_sign: 1.0
  use_imu_heading: true
  imu_topic: /imu/data_centered
  rpm_scale: 0.025
  servo_center: 0.5
  max_steer_rad: 0.5
